lib/bathlarp_web/controllers/v1/account_confirmation_controller.ex

```diff
@@ -44,7 +44,7 @@ defmodule BathLARPWeb.V1.AccountConfirmationController do
     }
 
   @spec create(Conn.t(), map()) :: Conn.t()
-  def create(conn = %Conn{}, %{}) do
+  def create(%Conn{} = conn, %{}) do
```

Comment:
This change (and others like it) are for consistency. The order doesn't really matter, but the linter likes the order to be consistent throughout the project, and the code that Phoenix generates for us uses this order. So this means less rewriting of generated code.